source/openstack/details.rst

```diff
@@ -0,0 +1,115 @@
+.. _openstack-details:
+
+OpenStack Details
+=================
+
+These are notes on how the OSUOSL is setting up OpenStack (as of May 2020).
+
+Summary
+-------
+
+We use Chef as our configuration management tool of choice and have created an environment using the `OpenStack Chef`_
+Cookbooks. The compute nodes are running CentOS 7 and the controller node is an x86 virtual machine running CentOS 7.
+All storage for the cluster is powered via a `Ceph`_ cluster backed by a 10Gb network.
+
+We have created our own site specific wrapper `OpenStack cookbook`_ that also includes any changes needed to make
+OpenStack run on the varying architectures. It currently requires some private cookbooks to run, however we are
+working on removing those requirements.
+
+.. _OpenStack Chef: https://docs.openstack.org/openstack-chef/latest/
+.. _OpenStack cookbook: https://github.com/osuosl-cookbooks/osl-openstack
+.. _Ceph: https://ceph.com/
+
+Hardware Stack (x86)
+--------------------
+
+- Controller (1)
+
+  - One (1) x86_64 Virtual Machine running on Ganeti with KVM
+  - 8g RAM / 10g disk / 8 vCPU
+
+- Compute (8)
+
+  - Eight (8) servers, each with:
+
+    - 2 x Intel(R) Xeon(R) CPU E5-2620 v3 @ 2.40GHz
+    - 128G RAM
+    - 10Gb NIC
+
+- Storage (8) - (shared between x86 and aarch64 clusters)
+
+  - Eight (8) Dell PowerEdge R710
+  - 64g RAM
+  - 6 x 4TB SATA / 1 x 800GB NVMe
+
+Hardware Stack (POWER)
+----------------------
+
+- Controller (1)
+
+  - One (1) x86_64 Virtual Machine running on Ganeti with KVM
+  - 24g RAM / 20g disk / 8 vCPU
+
+- Compute (9)
+
+  - Five (5) 8247-22L POWER8 systems w/ 512g RAM
+  - Four (5) 9006-12P POWER9 systems w/ 512g RAM
+
+- Storage (5)
+
+  - Five (5) 8001-22C POWER8 systems
+  - 256g RAM
+  - 8 x 8TB SATA / 4 x 240GB SSD
+
+POWER-Specific Changes
+~~~~~~~~~~~~~~~~~~~~~~
+
+- Turn off SMT
```

Comment:
Let's change this (and the bullet below) to the next smaller header size. I think it'll look better.